src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -825,6 +831,19 @@ export class DBTCoreProjectIntegration
     return modelPaths;
   }
 
+  private async findSeedPaths(): Promise<string[]> {
+    let seedPaths = await this.python.lock(
+      (python) => python`to_dict(project.config.seed_paths)`,
+    );
+    seedPaths = seedPaths.map((seedPath: string) => {
```

Comment:
some logs could be useful here. also you can directly return result of map instead of mutating `seedPaths`

Reply:
Agreed, I was thinking of that but it seems we already applied same code above. will change both cases.